javascript/javascript1/week2/event.js

```diff
@@ -0,0 +1,10 @@
+
+  function getEventWeekday (dayList) {
+      const weekDays = ['Sunday','Monday','Tuesday','Wednesday','Thursday','Friday','Saturday'];
```

Comment:
The code works fine as it is but by declaring `weekDays` outside the function it won't be redeclared everytime `getEvenWeekday` is executed.

Reply:
Got it. Declared outside.